frontend-react/src/components/EditableCompare.tsx

```diff
@@ -115,12 +85,13 @@ export const EditableCompare = forwardRef(
 
                     const threeParts = splitOn(modifiedText, start, end);
                     // we're using HTML5's <s> tag to show error, style sets background to red.
-                    const errorHtml = `${threeParts[0]}<s>${threeParts[1]}</s>${threeParts[2]}`;
+                    const errorHtml = `${threeParts[0]}<s data-testid="EditableCompare__errorDiff">${threeParts[1]}</s>${threeParts[2]}`;
```

Comment:
Added this test id so we can find it in tests: https://github.com/CDCgov/prime-reportstream/pull/7713/files#diff-cd109c0ecef22f0c55512e1aad07a73668fc622e5793c1e7fa1dc6aec2f7df6eR110